RadixWallet/Clients/HTTPClient/HTTPClient+Interface.swift

```diff
@@ -0,0 +1,9 @@
+// MARK: - HTTPClient
+public struct HTTPClient: Sendable, DependencyKey {
```

Comment:
Extracted HTTP loading logic from the GatewayClient so it can be reused for other purposes.